### DISCUSSION TITLE
Retrieving status-400 codes

### DISCUSSION THREAD

**Discussion by Batimius** [hoppscotch/hoppscotch#3309] [Q&A]
Hello there. So, I was testing a local application (an app running on `http://localhost:3000`), and in my application, I return a status-400 response when the user does not provide the appropriate data to the API endpoint. When running the requests through Hoppscotch, any request that would end up in a status-400 response would give me a "Could not send request" screen. It wouldn't give me the response itself, but it would rather just display that it could not send the request.

I am unsure whether this is an intended behavior or not. I am using the extension and I have whitelisted `http://localhost:3000`, so I am unsure why this is an issue. If there is a way for me to fix this, please let me know. I am running Hoppscotch as a PWA from hoppscotch.io. That is all. Thank you for your time and help, they are truly appreciated.

**Comment by fayaz07** [comment]
I have a similar question, @Batimius did you find any work-around? anyone from hoppscotch can help?

**Reply by Batimius** [reply]
Unfortunately not. Still having the same issues. I'm assuming that this might be an extension issue. Hoping they can fix it.

**Reply by fayaz07** [reply]
this is a basic functionality, I don't understand how can this go unseen

**Reply by Batimius** [reply]
Maybe it thinks that status-400 codes mean that the website does not exist. I'll open an issue.
